ArduCopter/mode_stabilize.cpp

```diff
@@ -5,19 +5,23 @@
  */
 
 // stabilize_init - initialise stabilize controller
-bool Copter::ModeStabilize::init(bool ignore_checks)
+bool Copter::ModeStabilize::ok_to_enter() const
 {
     // if landed and the mode we're switching from does not have manual throttle and the throttle stick is too high
     if (motors->armed() && ap.land_complete && !_copter.flightmode->has_manual_throttle() &&
             (get_pilot_desired_throttle(channel_throttle->get_control_in()) > get_non_takeoff_throttle())) {
         return false;
     }
-    // set target altitude to zero for reporting
-    pos_control->set_alt_target(0);
 
     return true;
```

Comment:
Shouldn't this be calling the base implementation?

Reply:
Yes.